docs/GettingStarted.md

```diff
@@ -2497,3 +2498,58 @@ However, additional instrumentation provided by Datadog can be activated alongsi
 | `OpenTracing::FORMAT_TEXT_MAP` | Yes        |                        |
 | `OpenTracing::FORMAT_RACK`     | Yes        | Because of the loss of resolution in the Rack format, please note that baggage items with names containing either upper case characters or `-` will be converted to lower case and `_` in a round-trip respectively. We recommend avoiding these characters or accommodating accordingly on the receiving end. |
 | `OpenTracing::FORMAT_BINARY`   | No         |                        |
+
+### Profiling
+
+*Currently available as BETA feature.*
```

Comment:
I think one could read this as: _"I have a brand new, low criticality service that I want to add `ddtrace` to. I guess I'll test the new beta profiling feature"_.

If I'm not mistaken, just following these instructions will not display profiling results in users Datadog dashboard, is that right? If so, we should clarify this statement with something along the lines: "... if you'd like to try it, contact Datadog support at ... so we can enable the beta in your account."

Reply:
I've just removed the docs inhttps://github.com/DataDog/dd-trace-rb/pull/1444 . This way we decouple "merging to master" with "actually releasing it".